HucaresReact/src/pages/MLP/MLPTable.jsx

```diff
@@ -6,6 +6,7 @@ import { chunkArray } from '../../utils/Array';
 import PaginationContainer from '../../components/Pagination/Pagination';
 import { MLPDeleteModal } from '../../components/Modal/DataHelpers/MLPHelpers/MLPDeleteModal';
 import { InfoEditingModal } from '../../components/Modal/InfoEditingModal';
+import Button from "../../../node_modules/semantic-ui-react/dist/commonjs/elements/Button/Button";
```

Comment:
What the hell is this and where you are using it?

Reply:
Legacy code, will remove